extensions/mssql/src/objectManagement/localizedConstants.ts

```diff
@@ -343,6 +343,7 @@ export const RestorePlanSectionText = localize('objectManagement.restoreDatabase
 export const RestoreFromBackupFileOptionText = localize('objectManagement.restoreDatabase.restoreFromBackupFileOptionText', "Backup file");
 export const RestoreFromDatabaseOptionText = localize('objectManagement.restoreDatabase.restoreFromDatabaseOptionText', "Database");
 export const RestoreFromUrlText = localize('objectManagement.restoreDatabase.restoreFromUrlText', "URL");
+export const RestoreFromS3UrlText = localize('objectManagement.restoreDatabase.restoreFromS3UrlText', "S3 URL");
```

Comment:
Should this have (Preview) on it? Not sure if we wanted to label subfeatures with preview, but given our earlier discussion might be worth it to do that until we're sure we aren't going to make any large changes anymore.

Reply:
The whole dialog is marked as preview in the command palette

Reply:
Oops, missed that! Looks good then